modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable sort-keys */
```

Comment:
I believe that you need to delete this line.

Reply:
done. thanks